maintenance-branch

```diff
@@ -1 +1 @@
-4-0-dev
+remove-monkey-patching
```

Comment:
Again, is this actually dependant on others?

Reply:
Only on Core.